terraform/scripts/setup.sh.tpl

```diff
@@ -1,5 +1,13 @@
 #!/bin/bash
 
+# Note that this is a Terraform template file (.tpl). Terraform replaces variables wrapped in curly braces after a dollar sign at runtime.
```

Comment:
This comment carries over to the generated shell script, when the comment no longer applies.  Is is possible to wordsmith so that it works in both cases?  Or maybe just to add a specific comment to the one place where we escape the dollar sign  ("$${} in a terraform template to interpret in bash rather than Terraform https://developer.hashicorp.com/terraform/language/expressions/strings#escape-sequences-1")